sources/csharp2/KalturaClient/Request/BaseRequestBuilder.cs

```diff
@@ -43,18 +34,17 @@ public string Boundary
         public BaseRequestBuilder(string service)
         {
             this.service = service;
-            bufferRead = new byte[BUFFER_SIZE];
-            requestData = new StringBuilder(System.String.Empty);
         }
 
         abstract public MultiRequestBuilder Add(IRequestBuilder requestBuilder);
         abstract public object Deserialize(XmlElement xmlElement);
 
-        private void Log(string msg)
+        private void Log(string msg, string requestId)
```

Comment:
requestId could be a protected member, no need to pass it through methods.

Reply:
done: 4025cc3858f2210933ab5c6759e40afab0680c4c